Szwalnia/Realizacja_proc_szcz.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace Szwalnia
+{
+    public partial class Realizacja_proc_szcz : Form
+    {
+        public Realizacja_proc_szcz(SzwalniaEntities db)
+        {
+            InitializeComponent();
+
+        }
+
+        private void btnRealProcSzcz_Click(object sender, EventArgs e)
```

Comment:
niepotrzebne skróty - za miesiąc nie będzie Pan pamiętał co to znaczyło nie mówiąc o osobach z innych zespołów, proszę zmienić na pełną nazwę - kod nie ma limitu na znaki...